regression-test/suites/point_query_p0/test_point_query_partition_IN.groovy

```diff
@@ -0,0 +1,139 @@
+    def realDb = "regression_test_serving_p0"
+    def tableName = realDb + ".tbl_point_query_partition_IN"
+    sql "CREATE DATABASE IF NOT EXISTS ${realDb}"
+    // Parse url
+    String jdbcUrl = context.config.jdbcUrl
+    String urlWithoutSchema = jdbcUrl.substring(jdbcUrl.indexOf("://") + 3)
+    def sql_ip = urlWithoutSchema.substring(0, urlWithoutSchema.indexOf(":"))
+    def sql_port
+    if (urlWithoutSchema.indexOf("/") >= 0) {
+        // e.g: jdbc:mysql://locahost:8080/?a=b
+        sql_port = urlWithoutSchema.substring(urlWithoutSchema.indexOf(":") + 1, urlWithoutSchema.indexOf("/"))
+    } else {
+        // e.g: jdbc:mysql://locahost:8080
+        sql_port = urlWithoutSchema.substring(urlWithoutSchema.indexOf(":") + 1)
+    }
+    // set server side prepared statement url
+    def prepare_url = "jdbc:mysql://" + sql_ip + ":" + sql_port + "/" + realDb + "?&useServerPrepStmts=true"
+
+    def generateString = {len ->
+        def str = ""
+        for (int i = 0; i < len; i++) {
+            str += "a"
+        }
+        return str
+    }
+
+    def nprep_sql = { sql_str ->
+        def url_without_prep = "jdbc:mysql://" + sql_ip + ":" + sql_port + "/" + realDb
+        connect(user = user, password = password, url = url_without_prep) {
+            sql sql_str
+        }
+    }
+
+    sql """DROP TABLE IF EXISTS ${tableName}"""
+    sql """
+              CREATE TABLE IF NOT EXISTS ${tableName} (
+                `k1` int(11) NULL COMMENT "",
+                `value` text NULL COMMENT ""
+              ) ENGINE=OLAP
+              UNIQUE KEY(`k1`)
+              PARTITION BY RANGE(`k1`)
+              (
+                  PARTITION `p1` VALUES LESS THAN ("1"),
+                  PARTITION `p2` VALUES LESS THAN ("10"),
+                  PARTITION `p3` VALUES LESS THAN ("30"),
+                  PARTITION `p4` VALUES LESS THAN ("40"),
+                  PARTITION `p5` VALUES LESS THAN ("1000")
+              )
+              DISTRIBUTED BY HASH(`k1`) BUCKETS 1
+              PROPERTIES (
+              "replication_allocation" = "tag.location.default: 1",
+              "store_row_column" = "true",
+              "enable_unique_key_merge_on_write" = "true",
+              "light_schema_change" = "true",
+              "storage_format" = "V2")
+        """
+
+    sql """INSERT INTO ${tableName} VALUES (1, 'a')"""
+    sql """INSERT INTO ${tableName} VALUES (2, 'b')"""
+    sql """INSERT INTO ${tableName} VALUES (-1, 'c')"""
+    sql """INSERT INTO ${tableName} VALUES (11, 'd')"""
+    sql """INSERT INTO ${tableName} VALUES (15, 'e')"""
+    sql """INSERT INTO ${tableName} VALUES (33, 'f')"""
+    sql """INSERT INTO ${tableName} VALUES (45, 'g')"""
+    sql """INSERT INTO ${tableName} VALUES (999, 'h')"""
+    def result1 = connect(user=user, password=password, url=prepare_url) {
+        def stmt = prepareStatement "SELECT * FROM ${tableName} WHERE k1 IN (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) ORDER BY k1"
```

Comment:
not add order by

Reply:
check the result use something like order_qt_xxx, maybe you should modify Suite.groovy to support something like qe_order_qt_xxx.